frontend/cypress/integration/featureFiles/waypoint.feature

```diff
@@ -13,6 +13,7 @@ Feature: Kiali Waypoint related features
   @waypoint
```

Comment:
Not related to this PR, but if the whole feature is already tagged with @waypoint, then it's not necessary to tag the scenarios as well, right?